foreign_cc/private/cmake_script.bzl

```diff
@@ -81,7 +93,7 @@ def create_cmake_script(
         for key in params.env
     ]
     str_cmake_cache_entries = " ".join([
-        "-D{}=\"{}\"".format(key, _escape_dquote_bash(params.cache[key]))
+        "-D{}=\"{}\"".format(key, _escape_dquote_bash_crosstool(params.cache[key]))
```

Comment:
Can you add a comment about what's happening with these `__var_{}` variables here?